// angular-tabs-component/__ivy_ngcc__/dist/angular-tabs-component.module.js
// Angular imports
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { FormsModule } from '@angular/forms';
// Components imports
import { TabComponent } from './components';
// Containers imports
import { TabsContainer } from './containers';
import * as ɵngcc0 from '@angular/core';
var TabModule = /** @class */ (function () {
    function TabModule() {
    }
    /** @nocollapse */
    TabModule.ctorParameters = function () { return []; };
TabModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: TabModule });
TabModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function TabModule_Factory(t) { return new (t || TabModule)(); }, imports: [[CommonModule, FormsModule]] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(TabModule, { declarations: function () { return [TabComponent,
        TabsContainer]; }, imports: function () { return [CommonModule, FormsModule]; }, exports: function () { return [TabComponent,
        TabsContainer]; } }); })();
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(TabModule, [{
        type: NgModule,
        args: [{
                imports: [CommonModule, FormsModule],
                declarations: [
                    TabComponent,
                    TabsContainer
                ],
                exports: [
                    TabComponent,
                    TabsContainer
                ]
            }]
    }], function () { return []; }, null); })();
    return TabModule;
}());
export { TabModule };
